stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -1507,6 +1511,69 @@ func _TFCOpSetAttrStringArray(_ op: CTFEOp,
   }
 }
 
+/// A TensorFlow device kind.
+public enum DeviceKind : UInt {
```

Comment:
Are we using its raw `UInt` value anywhere?  If not, it's better to remove the raw value type because it's _possible_ that cases are going to have associated values (e.g. `.gpu(0)`), in which case the enum will not be trivially `RawRepresentable`.